src/components/portal/__spec__.js

```diff
@@ -1,27 +1,74 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount, shallow } from 'enzyme';
 import Portal from './portal';
-import ReactPortal from 'react-portal';
+import Browser from '../../utils/helpers/browser';
 
 describe('Portal', () => {
-  let child, wrapper, reactPortal;
-
-  beforeEach(() => {
-    wrapper = shallow(
-      <Portal open>
-        <p className='child-element'></p>
-      </Portal>
-    );
+  let wrapper;
+  describe('when using default node', () => {
+    beforeEach(() => {
+      wrapper = shallow(
+        <Portal>
+          <p className='child-element' />
+        </Portal>
+      );
+    });
+
+    it('div with name portal should be created ', () => {
+      expect(wrapper.instance().node.classList[0]).toEqual('portal');
+    });
+
+    it('to match snapshot ', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
+
+    it('will unmount correctly', () => {
+      expect(wrapper.unmount());
```

Comment:
should you test that it cleans up the DOM?

Reply:
We should test what we expect it to do though?

Reply:
Sorry of course. This was an mistake. I was obviously tired on Friday. Oops.